library/Icingadb/Widget/RedundancyGroupStatistics.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/* Icinga DB Web | (c) 2024 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Widget;
+
+use Icinga\Chart\Donut;
+
+use Icinga\Module\Icingadb\Widget\Detail\ObjectStatistics;
+use ipl\Html\Text;
+use ipl\Html\ValidHtml;
+use ipl\Html\HtmlString;
+
+class RedundancyGroupStatistics extends ObjectStatistics
```

Comment:
Please rename this to *Object**s**Statistics*. There is not much which makes this redundancy group specific.

It's just not specific to hosts and services anymore. We will benefit from a generalized implementation somewhere else ;)